src/components/modal/__tests__/CdrModal.spec.js

```diff
@@ -35,34 +36,107 @@ describe('CdrModal.vue', () => {
         key: 'a'
       });
       await wrapper.vm.$nextTick();
-  
+
       wrapper.trigger('keydown', {
         key: 'Esc',
       });
       await wrapper.vm.$nextTick();
-  
+
       wrapper.trigger('keydown', {
         key: 'Escape',
       });
       await wrapper.vm.$nextTick();
-  
+
       expect(wrapper.emitted().closed.length).toBe(2);
     });
```

Comment:
Added a few unit tests below for better coverage - I didn't see where we had tests covering the different prop possibilities, let me know if I'm missing something though.